docs/.vitepress/theme/components/blog/BlogCard.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <h1>Blog</h1>
+
+  <ul>
+    <li>
+      <a :href="link"
+        ><b>{{ props.title }}</b></a
+      >
+      by {{ props.author }}.
```

Comment:
```suggestion
        ><b>{{ title }}</b></a
      >
      by {{ author }}.
```
You don't _need_ to reference `props` if you're using them in the template. And in this component, you only use `props` for the template, so you don't need to define `props`/assign the `defineProps` value to it.